tools/build_updated_packages.rb

```diff
@@ -59,7 +59,7 @@ def self.check_build_uploads(architectures_to_check = nil, name = nil)
   puts 'Skipping pip and gem remote update checks.'.orange
 else
   puts 'Checking for pip package version updates...'.orange
-  load 'tools/update_python_pip_packages.rb'
+  system 'tools/update_python_pip_packages.rb'
```

Comment:
Do you want to use Kernel.system since we override system?

Reply:
Good point. Fixed.